wordLadder.js

```diff
@@ -0,0 +1,74 @@
+/*
+A transformation sequence from word beginWord to word endWord using a dictionary wordList is a sequence of words beginWord -> s1 -> s2 -> ... -> sk such that:
+
+Every adjacent pair of words differs by a single letter.
+Every si for 1 <= i <= k is in wordList. Note that beginWord does not need to be in wordList.
+sk == endWord
+Given two words, beginWord and endWord, and a dictionary wordList, 
+return the number of words in the shortest transformation sequence from beginWord to endWord, or 0 if no such sequence exists.
+*/
+
+const { Queue } = require('./utils/queue');
+
+const NO_SEQUENCE_EXIST = 0;
+const UTF_16_LOWER_CASE_A = 97;
+const NUMBER_OF_ALPHABETS = 26;
+
+const getNeighbors = (word, wordListSet) => {
+    const neighbors = [];
+    const wordLength = word.length;
+
+    for (let i = 0; i < wordLength; i++) {
+        for (let j = 0; j < NUMBER_OF_ALPHABETS; j++) {
+            const sliceWordBeforeI = word.slice(0, i);
+            const sliceWordAfterI = word.slice(i + 1);
+            const currentLetter = String.fromCharCode(UTF_16_LOWER_CASE_A + j);
```

Comment:
This is okay, but every time you interact with ASCII you'll need to remember how to do this. I think making the `getAsciiFromChar` and `getCharFromAscii` functions makes life much easier.

Also, I have no idea what `UTF_16_LOWER_CASE_A + j` really represents, it would be nice if there was a clear variable for this.